sunpy/map/tests/test_mapbase.py

```diff
@@ -1728,6 +1729,22 @@ def test_draw_carrington_map(carrington_map):
     return fig
 
 
+def test_plot_settings_and_contour_params(simple_map):
```

Comment:
Is there is a small set of keywords this supports, I would like all of them tested if possible.